src/profile-logic/gecko-profile-versioning.js

```diff
@@ -1299,12 +1299,6 @@ const _upgraders = {
         display: ['marker-chart', 'marker-table', 'timeline-overview'],
         data: [{ key: 'name', label: 'Tick Reasons', format: 'string' }],
       },
-      {
-        // The schema is mostly handled with custom logic.
-        name: 'Network',
-        display: ['marker-table'],
-        data: [],
-      },
```

Comment:
Suggestion: I'm somewhat against this change since upgraders should be written once, and then never touched again. Any additional changes should only be in new profile upgraders. I think this change only makes the situation more confusing.

Reply:
I gave some thoughts to that before doing it. My initial thought was that the upgraders were broken, so could be fixed.

However:
* there's basically no drawback keeping this code (except that upgraded profiles would be different than new profiles in that regard). Everything works the same.
* there probably are existing profiles that were upgraded with this code, so removing it now could make it that we forget about them in the future.

So finally I think I'll do like you suggest.